cql3/statements/modification_statement.hh

```diff
@@ -256,6 +256,15 @@ public:
     future<std::vector<mutation>> get_mutations(query_processor& qp, const query_options& options, db::timeout_clock::time_point timeout, bool local, int64_t now, service::query_state& qs) const;
 
     virtual json_cache_opt maybe_prepare_json_cache(const query_options& options) const;
+
+    const std::vector<::shared_ptr<operation>>& get_column_operations() const {
+        return _column_operations;
+    }
+
+    const std::vector<lw_shared_ptr<column_condition>>& get_regular_conditions() const {
+        return _regular_conditions;
+    }
+
```

Comment:
I believe that if you have a specialized insert/update statement for group0 these won't be needed, because you'll be getting them during the prepare stage.